homeassistant/components/velux/__init__.py

```diff
@@ -19,12 +21,19 @@
 DOMAIN = "velux"
 DATA_VELUX = "data_velux"
 PLATFORMS = [Platform.COVER, Platform.LIGHT, Platform.SCENE]
+DEFAULT_SCAN_INTERVAL: int = 300  # Seconds
 _LOGGER = logging.getLogger(__name__)
 
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {vol.Required(CONF_HOST): cv.string, vol.Required(CONF_PASSWORD): cv.string}
+            {
+                vol.Required(CONF_HOST): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
+                vol.Optional(
+                    CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
+                ): cv.time_period,
+            }
```

Comment:
Sorry I am not completely clear if i understand this right.
I should change to config_flow in this PR? Is this not a separate PR?

Reply:
If the change to the schema is compulsory for this PR, then you should put it in draft, then make the config_flow PR, and when the config_flow PR is merged you can re-open this PR.
If the change the schema is optional, then you should remove that from the PR, and maybe it will be accepted.

Reply:
Thank you very much for explanation.
Currently i cannot spend time migrating to config_flow i have reverted the schema change.